docs/manual/system/settings.de.md

```diff
@@ -264,8 +264,11 @@ contao:
             - svgz
             - webp
 
-        # Contao automatically detects the best Imagine service out of Gmagick, Imagick and Gd (in this order). 
-        # To use a specific service, set its service ID here.
+        # As of Contao 4.6, Contao automatically detects the best Imagine service out of Gmagick, Imagick and Gd (in this order). 
+        # To use a specific service, set its service ID here:
```

Comment:
Hm, the automatic detection already exists in Contao 4.4 as well. Contao 4.6 offers to override the used service via this config.

Reply:
What about something like that?
```
        # Contao automatically detects the best Imagine service out of Gmagick, Imagick and Gd (in this order). 
        # As of Contao 4.6, to use a specific service, set its service ID here:
        # - Gmagick: "contao.image.imagine.gmagick"
        # - Imagick: "contao.image.imagine.imagick"
        # - Gd     : "contao.image.imagine.gd"
```

Reply:
Yes. However, since this is just the output of the reference dump, the changes should be suggested here: https://github.com/contao/contao/pull/806 😁